manifests/charts/istio-operator/files/gen-operator.yaml

```diff
@@ -211,3 +211,9 @@ spec:
               value: "300s"
             - name: REVISION
               value: ""
+          volumeMounts:
+          - mountPath: /tmp
```

Comment:
please use a different mount path then /tmp.

Reply:
@sdake The url fetcher by default extrats the chart tgz to `/tmp` dir
https://github.com/istio/istio/blob/83cfd682cdfd9a536a03531aab26174745087087/operator/pkg/helm/urlfetcher.go#L59-L61